zavod/zavod/meta/assertion.py

```diff
@@ -70,6 +71,20 @@ def parse_metrics(
                 yield from parse_filters(
                     Metric.ENTITY_COUNT, comparison, "country", value
                 )
+            case "property_values" if isinstance(value, dict) and value != {}:
+                for schema_name, props in value.items():
+                    for prop_name, threshold_value in props.items():
+                        yield Assertion(
+                            Metric.ENTITY_COUNT,
+                            comparison,
+                            threshold_value,
+                            "property_values",
+                            # We don't just put a Property.qname in the shape of "Schema:name" here because we want to
+                            # assert not on qnames, which are the canonical name of a property (e.g. Thing.country), but
+                            # on property values in entities of a specific type, like Company.country.
+                            # TODO(Leon Handreke): Maybe raise if the property does not exist on the Schema?
```

Comment:
yeah warning or error - I'm not sure I have a strong opinion, but some kind of validation/feedback would be amazing